platform/config.go

```diff
@@ -0,0 +1,110 @@
+package platform
+
+import (
+	"fmt"
+	"net/http"
+
+	"time"
+
+	"github.com/Peripli/service-broker-proxy/pkg/platform"
+	"github.com/cloudfoundry-community/go-cfclient"
+	"github.com/spf13/viper"
+)
+
+type RegistrationDetails struct {
+	User     string
+	Password string
+}
+
+func (rd RegistrationDetails) String() string {
+	return fmt.Sprintf("User: %s Host: %s", rd.User)
+}
+
+type PlatformClientConfiguration struct {
+	*cfclient.Config
+
+	Reg        *RegistrationDetails
+	createFunc func(*cfclient.Config) (*cfclient.Client, error)
+}
+
+var _ platform.ClientConfiguration = &PlatformClientConfiguration{}
+
+func (c *PlatformClientConfiguration) CreateFunc() (platform.Client, error) {
```

Comment:
Why this method is `CreateFunc`, there is another one on the same structure starting with lower case. It is hard to understand the code.

Reply:
I think that the createFunv (lower case) is leftover from refactoring